articles/azure/azs-how-deploy-sql-template-powershell.md

```diff
@@ -70,42 +70,44 @@ SQL Always On Repository - [sql-2016-ha](https://github.com/Azure/AzureStack-Qui
   git init
   git config core.sparseCheckout true
   git remote add origin -f "https://github.com/Azure/AzureStack-QuickStart-Templates"
-  "sql-2016-ha/*" | Out-File -FilePath .git/info/sparse-checkout -Encoding ASCII
+  "sql-2016-alwayson*" | Out-File -FilePath .git/info/sparse-checkout -Encoding ASCII
   cat .git/info/sparse-checkout
   git checkout master
   ```
 
 ## What does it deploy?
 
-This template deploys two SQL Server 2016 SP1 Enterprise / Standard / Developer instances in the Always On Availability Group using the PowerShell DSC Extension. It creates the following resources:
+This template deploys two SQL Server 2016 SP1 or SP2 Enterprise / Standard / Developer instances in the Always On Availability Group using the PowerShell DSC extension. It creates the following resources:
 
-- A network security group
+- A network security group.
 
-- A Virtual Network
+- A virtual network.
 
-- Four Storage Accounts (One for AD, One for SQL, One for File Share witness and One for VM diagnostic)
+- Four storage accounts (One for AD, One for SQL, one for file share witness and one for VM diagnostic).
 
-- Four public IP address (One for AD, Two for each SQL VM and One for Public LB bound to SQL Always On Listener)
+- Four public IP address (One for AD, Two for each SQL VM and one for public LB bound to SQL always on listener).
 
-- One external load balancer for SQL VMs with Public IP bound to SQL always On Listener
+- One external load balancer for SQL VMs with public IP bound to the SQL always on listener.
 
-- One VM (WS2016) configured as Domain Controller for a new forest with a single domain
+- One VM (WS2016) configured as domain controller for a new forest with a single domain.
 
-- Two VM (WS2016) configured as SQL Server 2016 SP1 Enterprise / Standard / Developer and clustered.
+- Two VM (WS2016) configured as SQL Server 2016 SP1 or SP2 enterprise/standard/developer and clustered.
```

Comment:
If Enterprise, Standard and Developer are actual product names, it's better to keep the capitalisation as it was